guides/common/modules/proc_adding-a-vnc-connection-for-a-managed-host-on-proxmox.adoc

```diff
@@ -0,0 +1,43 @@
+[id="Adding_a_VNC_Connection_for_a_Managed_Host_on_Proxmox_{context}"]
+= Adding a VNC Connection for a Managed Host on Proxmox
```

Comment:
I would call it Remote Console instead of VNC in all sections which are foreman specific.
In the end, the connection from foreman to proxmox its using VNC. But the user does only press the "Console" button and not the "VNC" button.